cmd/daemon/main.go

```diff
@@ -23,14 +34,23 @@ func main() {
 		log.Fatal(err)
 	}
 
-	ctrd, err := freezer.Connect()
+	var freezeThaw daemon.FreezeThawer
+	switch runtimeType {
+	case RuntimeTypeDocker:
+		freezeThaw, err = docker.NewDockerService()
```

Comment:
might be nice to make both methods be called `New` or `Connect` so they match. Either way ideally we wouldn't have `docker.NewDockerService` because it stutters

Reply:
Both `New` sounds like a good idea